.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+
+If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.
```

Comment:
Some would disagree with that one, as it does not apply to a lot of use cases where practical diffs of PRs show a lot more than thousand pages of theoretical RFC discussion :D